src/artefacts_demo_control/test/artefacts_demo_control/collision_test.py

```diff
@@ -63,13 +65,11 @@ def generate_test_description():
 
 
 class TestCollision(unittest.TestCase):
-    def test_collsion(self):
+    def get_box_location(self):
```

Comment:
This function should probably be outside of the TestCollision class.
hint: `self` is never used.
Moreover, I can imagine it being generalized a bit (to get location of an object by name/id) later, and become part of some util library

Reply:
Done, the get_box_location function is now outside the class, has been renamed to get_model_location, and now takes a model name argument (In our case artefacts_box).